src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -441,6 +449,20 @@ impl HummockEventHandler {
                     UploaderEvent::DataSpilled(staging_sstable_info) => {
                         self.handle_data_spilled(staging_sstable_info);
                     }
+
+                    UploaderEvent::ImmMerged(merge_output) => {
+                        // update read version for corresponding table shards
+                        let read_guard = self.read_version_mapping.read();
+                        read_guard.get(&merge_output.table_id).map_or((), |shards| {
```

Comment:
Shall we panic or add a log if the instance id is not found?

Reply:
If we cannot find the corresponding instance when we handle `ImmMerged`, the merged imm is useless. It is safe to drop it silently.